crates/bevy_ecs/src/schedule/ambiguity_detection.rs

```diff
@@ -0,0 +1,284 @@
+use crate::component::ComponentId;
+use crate::schedule::{AmbiguityDetection, SystemContainer, SystemStage};
+use crate::world::World;
+
+use bevy_utils::HashMap;
+use fixedbitset::FixedBitSet;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub enum AmbiguityReportLevel {
+    Off,
+    Minimal,
+    Verbose,
+}
```

Comment:
It might be nice to have the documentation here as well (if you're okay with the partial duplication).